### PR TITLE
Added new command "okta-iam-list-user-applications"

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33530

## Description
Added new command: "okta-iam-list-user-applications"
Added new query filter for update user use case

## Minimum version of Demisto
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Shellyber** [comment]
@roysagi Please review the release notes

File: Packs/Okta/ReleaseNotes/2_1_7.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+#### Classifiers
+##### Okta IAM - App Sync (classifier)
+- Added classification for Okta log - update profile event (used for Identity and Lifecycle management).
```
Comment (roysagi) [inline Packs/Okta/ReleaseNotes/2_1_7.md:4]:
```suggestion
- Added the classification for Okta log update user profile events to **IAM - App Update** incident type (used for the **Identity Lifecycle Management (ILM)** pack).
```

File: Packs/Okta/ReleaseNotes/2_1_7.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+#### Integrations
+##### Okta IAM
+- Added new command: *okta-iam-list-user-applications*
+- Added support for update user in fetch query.
```
Comment (roysagi) [inline Packs/Okta/ReleaseNotes/2_1_7.md:9]:
```suggestion
- Added support for fetching update user profile events.
```

File: Packs/Okta/Scripts/AssignedAppsTableWidget/AssignedAppsTableWidget.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  id: AssignedAppsTableWidget
+  version: -1
+enabled: false
+name: AssignedAppsTableWidget
+script: '-'
+subtype: python3
+system: false
+tags:
+- Widget
+- ' IAM'
```
Comment (dantavori) [inline Packs/Okta/Scripts/AssignedAppsTableWidget/AssignedAppsTableWidget.yml:14]: do we use this tag anywhere else?

File: Packs/Okta/Scripts/AssignedAppsTableWidget/AssignedAppsTableWidget.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- ' dynamic-indicator-section'
```
Comment (dantavori) [inline Packs/Okta/Scripts/AssignedAppsTableWidget/AssignedAppsTableWidget.yml:15]:
```suggestion
- 'dynamic-indicator-section'
```

File: Packs/Okta/ReleaseNotes/2_1_7.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+
+#### Classifiers
+##### Okta IAM - App Sync (classifier)
+- Added the classification for Okta log update user profile events to **IAM - App Update** incident type (used for the **Identity Lifecycle Management (ILM)** pack).
```
Comment (dantavori) [inline Packs/Okta/ReleaseNotes/2_1_7.md:4]:
```suggestion
- Added a classification from user profile update events in Okta to the **IAM - App Update** incident type (used in the **Identity Lifecycle Management** pack).
```

File: Packs/Okta/ReleaseNotes/2_1_7.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Integrations
+##### Okta IAM
+- Added command: ***okta-iam-list-user-applications***.
```
Comment (dantavori) [inline Packs/Okta/ReleaseNotes/2_1_7.md:8]:
```suggestion
- Added the ***okta-iam-list-user-applications*** command.
```

**Review by dantavori — CHANGES_REQUESTED** [review]

File: Packs/Okta/ReleaseNotes/2_1_7.md (hunk 6 above)
Comment (Shellyber) [reply Packs/Okta/ReleaseNotes/2_1_7.md:8]: This is after rn review. Don't think we should change it

**Review by dantavori — APPROVED** [review]

**Pull request merged**